strapi/strapi
# Add seed data and schemas to CLI e2e app template

### What does it do?

Adds a complex test template for CLI e2e tests

### Why is it needed?

- needed to validate that the data transfer tool used for test setup is trustworthy by verifying every record expected
- can be used by CLI tests as a quick way to have every Strapi data structure available

### How to test it?

- run `yarn test:cli` and the test app should generate with this structure (content types)
- run the `cli-e2e/scripts/dts-import.js` targeting the data/complex.tar.gz and it should load into the project

Note: the automatic seeding will come in a later PR with a utility function for tests to use to call the import script

### Related issue(s)/PR(s)
